fwoa-tools/config/jest.config.json

```diff
@@ -3,10 +3,10 @@
   "collectCoverage": true,
   "coverageThreshold": {
     "global": {
-      "branches": 0,
-      "functions": 0,
-      "lines": 0,
-      "statements": 0
+      "branches": 60,
+      "functions": 80,
+      "lines": 80,
+      "statements": 80
     }
```

Comment:
Have we verified that these numbers pass on our current code?

Reply:
Yes, I've verified locally that it passes